get-started/next-steps-resources.md

```diff
@@ -0,0 +1,178 @@
+# Next steps & resources
+
+We've designed our docs to help you find the information you need quickly and efficiently, whether you're a new user or an experienced professional.
+
+## Information architecture
```

Comment:
I this reads as docs-about-docs, but I wish we could make it more use case / audience-centric

can we also have links here? sucks to see the section I want and not be able to click.

Reply:
Done!